test/functional/test_framework/wallet.py

```diff
@@ -119,13 +121,14 @@ def _bulk_tx(self, tx, target_weight):
         """Pad a transaction with extra outputs until it reaches a target weight (or higher).
         returns the tx
         """
-        tx.vout.append(CTxOut(nValue=0, scriptPubKey=CScript([OP_RETURN, b'a'])))
+        tx.vout.append(CTxOut(nValue=0, scriptPubKey=CScript([OP_RETURN])))
         dummy_vbytes = (target_weight - tx.get_weight() + 3) // 4
```

Comment:
Hmm from the way I understand this, addition of 3 is helpful in the case when `(target_weight - tx.get_weight() < 4` because otherwise the floor division by 4 will cause no padding to be added and the transaction will end up being same in vbytes. Is this the right way to understand this?

Tying this to an earlier comment of mine - if `target_weight` had been called `target_weight_wu`, the code would become more self explanatory.

I feel this is worth adding as a comment for reference in the code, and thanks for the explanation!

Reply:
> To my knowledge, weight is always measured in weight units, so I don't think adding the _wu suffix here would add any new information.